serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/Database.kt

```diff
@@ -0,0 +1,29 @@
+package xyz.room409.serif.serif_shared
+import xyz.room409.serif.serif_shared.db.*
+import xyz.room409.serif.serif_shared.db.DriverFactory
+
+object Database {
```

Comment:
This just seems to pass everything through, can we just have the implementation here?

Reply:
It does just pass through, really the key difference is the package the file is a part of. This Database file is part of the `xyz.room409.serif.serif_shared` package, while the other file is part of the `xyz.room409.serif.serif_shared.db` package, along with all of the generated code from sqldelight. My initial thought was have the methods for interfacing with the generated code all tucked away in the one file that was in the `.db` package (Db.kt), and then provide `shared_serif` with an interface to it (Database.kt), but we already have 2 pass throughs happening with the state and client functions and there wasn't any additional cleanup needing to be done between the generated code and the shared code so moving the implementation up one level makes sense.